src/Microsoft.OData.Core/Build.Net35/Microsoft/OData/Core/UriParser/Aggregation/BaseTokens.cs

```diff
@@ -18,4 +18,11 @@ namespace Microsoft.OData.UriParser.Aggregation
     internal abstract class ApplyTransformationToken : QueryToken

```

Comment:
Even though these classes are super tiny, I prefer 1 file per class.  It becomes difficult to tell if a class should be in BaseTokens.cs based on if it is a root base class or an intermediate base class.